tests/unit/views/services/camera_test.js

```diff
@@ -0,0 +1,112 @@
+import Rx from 'rxjs';
+import React from 'components/react';
+
+import CameraServiceView from 'js/views/services/camera';
+
+// See capabilities at https://facebook.github.io/react/docs/test-utils.html
+const TestUtils = React.addons.TestUtils;
+
+function waitFor(fn) {
```

Comment:
I think I like it and it works well when you're used to work with MarionetteJS.
I also think it should be moved to a separate file, but we can also wait until we need it somewhere else.